packages/sdks/snippets/angular/src/app/product-details/product-details.component.ts

```diff
@@ -0,0 +1,43 @@
+import { CommonModule } from '@angular/common';
+import { Component } from '@angular/core';
+import { fetchOneEntry, type BuilderContent } from '@builder.io/sdk-angular';
+
+@Component({
+  selector: 'app-product-details',
+  standalone: true,
+  imports: [CommonModule],
+  template: `
+    <div *ngIf="productDetails">
+      <h1>{{ productDetails.data?.['name'] }}</h1>
+      <img
+        [src]="productDetails.data?.['image']"
+        [alt]="productDetails.data?.['name']"
+        width="400"
+        height="500"
+      />
+      <p>{{ productDetails.data?.['collection'].value.data.copy }}</p>
+      <p>
+        Price:
+        {{ productDetails.data?.['collection'].value.data.price }}
```

Comment:
any reason why we're using `[]` bracked notation here? 
```suggestion
      <h1>{{ productDetails.data?.name }}</h1>
      <img
        [src]="productDetails.data?.image"
        [alt]="productDetails.data?.name"
        width="400"
        height="500"
      />
      <p>{{ productDetails.data?.collection.value.data.copy }}</p>
      <p>
        Price:
        {{ productDetails.data?.collection.value.data.price }}
```